### PR TITLE
Fixed to two issues.

### LINKED ISSUE #71: After login with 2FA, the url of feed page is something wrong.
The url of feed page is '/login_totp'. It should be '/feeds'.

### LINKED ISSUE #72: Some functions move to stip-common.
- get_text_field_value
- login
- login_totp

Those functions are duplicated, so should be moved to stip-common.

### PR DESCRIPTION
**Pull request by stmtstk**
Correspond to #71  and #72 .

### CODE REVIEW

File: src/core/views.py [hunk 1]
```diff
@@ -1,84 +1,45 @@
 import os
 import json
-import pyotp
+# import pyotp
```
Comment (tkhsknsk) [inline src/core/views.py]: There is a comment out left.

**Review by tkhsknsk — DISMISSED** [review]
Verified that it moved to stip-common.

File: src/core/views.py (hunk 1 above)
Comment (stmtstk) [reply src/core/views.py]: Thanks.
I removed a comment and push the branch.
Please re-review and approve it.

**Review by tkhsknsk — APPROVED** [review]
OK.

**Pull request merged**